releasenotes/notes/1.1/add-annotated-arg-to-power-4afe90e89fa50f5a.yaml

```
@@ -1,5 +1,5 @@
 ---
-features:
+upgrade_circuits:
```

The upgrade section is normally reserved to document changes that might require an end user to make a change in their usage when upgrading from 1.0.x -> 1.1.0. In this case the existing behavior is unchanged this is a new flag on the `power()` method that can be used, but any existing use of it wouldn't change.

```suggestion
features_circuits:
```